File: MODUL 4/PRAK405-2410817210005-Muhammad_Irgi_Fahrezha.c
#include <stdio.h>

int main() {
    int n;
    int kelipatan;
    int total = 0;
    int baris_total = 0;
    
    
    scanf("%d", &n);
    scanf("%d", &kelipatan);
   
    for (int i = 1; i <= n; i++) {
        for (int j = 1; j <= i; j++) {
            baris_total += j * kelipatan;
            printf("(%d * %d)", j, kelipatan);

            if (j < i) {
                printf(" + ");
            }
        }
        total += baris_total;
        printf(" = %d\n", baris_total);
    }
    printf("%d\n", total);

    return 0;
}

File: MODUL 4/PRAK405-2410817210005-Muhammad_Irgi_Fahrezha.py
n, kelipatan = input("").split()
n = int(n)
kelipatan = int(kelipatan)

total = 0

for i in range(1, n + 1):
    baris_total = 0
   
    for j in range(1, i + 1):
        baris_total += j * kelipatan
        print(f"({j} * {kelipatan})", end="")
        if j < i:
            print(" + ", end="")
    total += baris_total
    print(f" = {baris_total}")


print(total)